code/typescript/pipelines-workshop/test/hitcounter.test.ts

```diff
@@ -48,18 +45,3 @@ test('Lambda Has Environment Variables', () => {
         }
     }));
 });
-
-test('Read Capacity can be configured', () => {
-    const stack = new cdk.Stack();
-
-    expect(() => {
-        new HitCounter(stack, 'MyTestConstruct', {
-            downstream: new lambda.Function(stack, 'TestFunction', {
-                runtime: lambda.Runtime.NODEJS_14_X,
-                handler: 'lambda.handler',
-                code: lambda.Code.inline('test')
-            }),
-            readCapacity: 3
-        });
-    }).toThrowError(/readCapacity must be greater than 5 and less than 20/);
-});
```

Comment:
Removed this test since `readCapacity` is no longer a parameter for `HitCounter` and causes Typescript to throw a ts error.